Dockerfile.dapper

```diff
@@ -74,7 +74,7 @@ ENV DAPPER_RUN_ARGS --privileged --network host -v /run/containerd/containerd.so
 ENV GO111MODULE off
 ENV DAPPER_ENV REPO TAG DRONE_TAG DRONE_BRANCH CROSS GOPROXY PUSH RKE2_IMAGE_REPO CODECOV_TOKEN
 ENV DAPPER_SOURCE /go/src/github.com/harvester/harvester/
-ENV DAPPER_OUTPUT ./bin ./dist ./package
+ENV DAPPER_OUTPUT ./bin ./deploy ./dist ./package ./pkg
```

Comment:
@votdev, may you also add `./api` here? So users can run `make generate` and get latest `swagger.json`.